Bottom: 201ed5320b47fcd7f111d95b48398b2bb25fa1c6
Top:    f91e709d2436c09a357dc294064e803e0caae2f8
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 14:20:16 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 31a6a73..46289f0 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -85,8 +85,8 @@ pthread_attr_t th_attr_detached;
 struct master_readmap_cond mrc = {
 	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, NULL, 0, 0, 0, 0};
 
-/*struct startup_cond suc = {
-	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, 0};*/
+struct startup_cond suc = {
+	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, 0};
 
 struct shutdown_cond fc = {
 	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, 0, 0};
